src/test/java/edu/kit/datamanager/repo/util/validators/DOIValidatorTest.java

```diff
@@ -44,11 +43,7 @@ public void valid() {
 
     @Test
     public void anotherValid() {
-        try {
-            assertTrue(validator.isValid("https://www.doi.org/10.1109/5.771073"));
-        } catch (Exception e){
-            fail(e.getMessage());
-        }
+        assertTrue(validator.isValid("https://www.doi.org/10.1109/5.771073"));
```

Comment:
You may add other valid entries here.
10.456/ldkjdeaIE
10./1?
.../.?
10.11.12/abcde
12345678901234567890/1?
Try to test the minimum and the maximum of possible characters 
Also try invalid characters: öß%*...